echo-telemetry/src/main/java/com/netflix/spinnaker/echo/config/TelemetryConfig.java

```diff
@@ -69,13 +70,31 @@ private OkClient telemetryOkClient(TelemetryConfigProps configProps) {
   @ConfigurationProperties(prefix = "telemetry")
   public static class TelemetryConfigProps {
 
+    public static final String DEFAULT_DEPLOY_METHOD = "none";
```

Comment:
I'd probably prefer a `null` here rather than `none` - mostly because then the backend queries will have to distinguish `none` from `null` without much of a difference.